react/TabbedShelf/tabbedShelf.css

```diff
@@ -0,0 +1,77 @@
+.container {
+  display:block;
+}
+
+.headline {}
+
+.container .mh7-ns,
```

Comment:
Is this a customization of their shelf? I think we shouldn't do this. What do you think @lbebber ?

Reply:
ideally no, but considering our time is short, and as long as this affects only the `tabbedShelf`, I think it's alright for now

Reply:
removed